lambda-functions/Lambda/psycopg2-no-SQLAlchemy/postgres_makeconn.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import logging
+import psycopg2
+from access_psycopg2 import make_conn, fetch
+
+# Lambda function logic for accessing the PostgreSQL DB in RDS
+# Select all data from table
+
+def lambda_handler(event, context):
+    SQL_cmd = "SELECT COUNT(\*) FROM reddit-comments"
```

Comment:
This SQL statement should probably be it's own file, and read into each python script that needs it

Reply:
Got it, will change this